beta/src/pages/apis/usestate.md

```diff
@@ -87,12 +87,16 @@ function handleClick() {
 
 `set` functions do not have a return value.
 
-#### Caveats {/*caveats*/}
+#### Caveats {/*setstate-caveats*/}
 
 * The `set` function only updates the state variable for the next render. If you read the state variable after calling the `set` function, you will still get the value that was on the screen before your call.
```

Comment:
How do you feel about adding an emphasis?

```suggestion
* The `set` function only updates the state variable for the _next_ render. If you read the state variable after calling the `set` function, you will still get the value that was on the screen before your call.
```

Maybe we could link the last subsentence to the learn section where we explain it with an example? Con: kinda makes the text lengthier. Pro: helps people see it themselves.